src/proposals/20250724/SparkEthereum_20250724.t.sol

```diff
@@ -0,0 +1,191 @@
+        assertEq(config.liquidationThreshold, 40_00);
+
+        executeAllPayloadsAndBridges();
+
+        ReserveConfig[] memory allConfigsAfter = createConfigurationSnapshot('', ctx.pool);
+
+        config.liquidationThreshold = 35_00;
+
+        _validateReserveConfig(config, allConfigsAfter);
+    }
+
+    function test_BASE_morphoRateLimitChanges() public onChain(ChainIdUtils.Base()) {
+        bytes32 morphoUsdcVaultDepositKey = RateLimitHelpers.makeAssetKey(
+            ForeignController(Base.ALM_CONTROLLER).LIMIT_4626_DEPOSIT(),
+            Base.MORPHO_VAULT_SUSDC
+        );
+
+        _assertRateLimit(morphoUsdcVaultDepositKey, 50_000_000e6, 25_000_000e6 / uint256(1 days));
+
+        executeAllPayloadsAndBridges();
+
+        _assertRateLimit(morphoUsdcVaultDepositKey, 100_000_000e6, 50_000_000e6 / uint256(1 days));
+    }
+
+    function test_ETHEREUM_transferEthToMultisig() public onChain(ChainIdUtils.Ethereum()) {
+        uint256 balanceBefore = IERC20(Ethereum.WETH_ATOKEN).balanceOf(Ethereum.TREASURY);
+
+        assertGe(balanceBefore, 500e18);
+
+        executeAllPayloadsAndBridges();
+
+        assertEq(IERC20(Ethereum.WETH_ATOKEN).balanceOf(Ethereum.TREASURY),    0);
+        assertEq(IERC20(Ethereum.WETH_ATOKEN).balanceOf(LIQUIDATION_MULTISIG), balanceBefore);
+    }
+
+    function test_ETHEREUM_transferTokensToGrove() public onChain(ChainIdUtils.Ethereum()) {
+        uint256 buidlIbalanceBefore = IERC20(Ethereum.BUIDLI).balanceOf(Ethereum.ALM_PROXY);
+        uint256 jtrsyBalanceBefore  = IERC20(Ethereum.JTRSY).balanceOf(Ethereum.ALM_PROXY);
+
+        assertGe(buidlIbalanceBefore, 800_000_000e6);
+        assertGe(jtrsyBalanceBefore,  370_000_000e6);
+
+        executeAllPayloadsAndBridges();
+
+        assertEq(IERC20(Ethereum.BUIDLI).balanceOf(Ethereum.ALM_PROXY), 0);
+        assertEq(IERC20(Ethereum.JTRSY).balanceOf(Ethereum.ALM_PROXY),  0);
+        assertEq(IERC20(Ethereum.BUIDLI).balanceOf(GROVE_ALM_PROXY),    buidlIbalanceBefore);
+        assertEq(IERC20(Ethereum.JTRSY).balanceOf(GROVE_ALM_PROXY),     jtrsyBalanceBefore);
+    }
+
+    function test_ETHEREUM_morpho_PTSPKUSDS25SEP2025Onboarding() public onChain(ChainIdUtils.Ethereum()) {
+        _testMorphoCapUpdate({
+            vault: SPARK_USDS_VAULT,
+            config: MarketParams({
+                loanToken:       Ethereum.USDS,
+                collateralToken: PT_SPK_USDS_25SEP2025,
+                oracle:          PT_SPK_USDS_25SEP2025_PRICE_FEED,
+                irm:             Ethereum.MORPHO_DEFAULT_IRM,
+                lltv:            0.965e18
+            }),
+            currentCap: 0,
+            newCap:     500_000_000e18
+        });
+        _testMorphoPendlePTOracleConfig({
+            pt:        PT_SPK_USDS_25SEP2025,
+            loanToken: Ethereum.USDS,
+            oracle:    PT_SPK_USDS_25SEP2025_PRICE_FEED,
+            discount:  0.15e18,
+            maturity:  1758758400  // Friday, September 26, 2025 12:00:00 AM UTC
```

Comment:
Are we sure this is Friday? My computer is showing Thursday UTC:

```
❯ date -u -d @1758758400
Thu Sep 25 00:00:00 UTC 2025
```

Reply:
u are right its thursday, changing